BrainCloudClient/Assets/BrainCloud/Client/BrainCloud/BrainCloudFile.cs

```diff
@@ -71,6 +71,47 @@ public BrainCloudFile(BrainCloudClient client)
 #endif
         }
 
+        
+        /// <summary>
+        /// Prepares a user file upload from memory. On success the file will begin uploading
+        /// to the brainCloud server.To be informed of success/failure of the upload
+        /// register an IFileUploadCallback with the BrainCloudClient class.
+        /// </summary>
+        /// <param name="cloudPath">The desired cloud path of the file</param>
+        /// <param name="cloudFilename">The desired cloud fileName of the file</param>
+        /// <param name="shareable">True if the file is shareable</param>
+        /// <param name="replaceIfExists">Whether to replace file if it exists</param>
+        /// <param name="localPath">The path and fileName of the local file</param>
```

Comment:
this should be fileData I believe... may also be worth explaining what fileData they're passing in here... Maybe some hints on how to get the appropriate fileData would help

Reply:
Addressed and fixed